frontend/src/UnitCounter.tsx

```diff
@@ -60,3 +54,30 @@ export function UnitCounter({
         </Stack>
     );
 }
+
+function Cost(props: { cost: number }) {
+    return (
+        <Stack
+            direction="row"
+            alignItems="center"
+            height="44px"
+            overflow="hidden"
```

Comment:
Recommend taking out this `overflow: hidden` to prevent losing the cost label at small window sizes!

| branch `cost-icons` | branch `main` |
|-|-|
| ![image](https://github.com/user-attachments/assets/66e9f39d-61f4-49c6-9e45-5d7c51ff5b1d) | ![image](https://github.com/user-attachments/assets/dd6315ec-ddeb-4f19-a3a0-123673ae7237) |

The CSS reason is murky to me, but when there isn't enough space for all elements to display at their intended size, there's some kind of prioritizing done on where to steal extra space from. So it must be that adding the `overflow: hidden` here communicates space can be stolen from this element.

Reply:
Cool, thanks!